src/models-api.ts

```diff
@@ -2,16 +2,14 @@ import OpenAI from "openai";
 
 // Model is the structure of a model in the model catalog.
 export interface Model {
-  id: string;
   name: string;
-  friendly_name: string;
-  model_version: number;
+  displayName: string;
+  version: number;
```

Comment:
I believe `version` comes back as a string from the new endpoint. We'll likely want to make sure we treat it the same as we've gotten versions like `2024-05-13` in the past (relevant [dotcom PR](https://github.com/github/github/pull/340823))

```suggestion
  version: string;
```

Reply:
Thank you, good catch! I've verified this doesn't break any of the existing flows.